### PR TITLE
feat: Overview Page

### LINKED ISSUE #52: Overview Page
Implement the `overview` page, make sure it's mobile responsive as well.

**Description:**

[Desktop View](https://www.figma.com/design/JW1DKwDBVJq97whSwlWDLS/Auto-swapper?node-id=356-11118&node-type=frame&t=I0aoi95acvIs1Imm-0)
[Mobile View](https://www.figma.com/design/JW1DKwDBVJq97whSwlWDLS/Auto-swapper?node-id=627-1729&node-type=frame&t=I0aoi95acvIs1Imm-0)

**Expected Behavior:**

Develop the Overview page based on the provided design.

**ETA**
This issue is expected to be completed within 2 days of assignment. Failure to create a pull request (or draft pull request) within the expected time will lead to being unassigned.

### PR DESCRIPTION
**Pull request by melnikga**
Added overview page for desktop devices

## Description

The Overview page has been added for desktop devices. I am still working on this task and will add the mobile version soon.

## Related Issue(s) #52 

Closes #52
## Checklist:
- [x] Desktop view
- [x] Mobile view
- [x] Read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [x] Verified this is not a duplicate of any existing pull request

- [x] Performed a self-review of the code
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Linked the issues which this PR resolves
- [x] Documented the changes in code (API docs will be generated automatically)

### CODE REVIEW

**Comment by melnikga** [comment]
I'm done with the desktop version, I'm still working on the mobile version.

**Review by princeibs — CHANGES_REQUESTED** [review]
Hi @melnikga, can you fix the build issue?

**Comment by melnikga** [comment]
@princeibs sure, I'll fix it now.

File: app/overview/page.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    { id: 2, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+    { id: 3, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+    { id: 4, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: false },
+    { id: 5, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+  ];
+
+  const handlePercentageChange = (e: React.ChangeEvent<HTMLSelectElement>, id: number) => {
+    console.log(e, id);
+  };
+
```
Comment (Abeeujah) [inline app/overview/page.tsx:27]: Hi @melnikga could you ensure the state is being updated anytime the user clicks on the percentage select field.

**Review by Abeeujah — CHANGES_REQUESTED** [review]

File: app/overview/page.tsx (hunk 1 above)
Comment (melnikga) [reply app/overview/page.tsx:27]: Hi @Abeeujah I'll fix it within a couple of hours

Comment (melnikga) [reply app/overview/page.tsx:38]: @Abeeujah I fixed that

**Comment by Oshioke-Salaki** [comment]
Build fails. Please fix

**Comment by melnikga** [comment]
@Oshioke-Salaki oh, i'm sorry, i'll fix that tonight

**Review by princeibs — APPROVED** [review]

**Pull request merged**